common/lib/xmodule/xmodule/vertical_block.py

```diff
@@ -76,7 +117,8 @@ def student_view(self, context):
             'unit_title': self.display_name_with_default if not is_child_of_vertical else None,
             'show_bookmark_button': child_context.get('show_bookmark_button', not is_child_of_vertical),
             'bookmarked': child_context['bookmarked'],
-            'bookmark_id': u"{},{}".format(child_context['username'], unicode(self.location))
+            'bookmark_id': u"{},{}".format(child_context['username'], unicode(self.location)),
+            'completables': self.get_completables_for_user(child_context['username']),
```

Comment:
How about calling this `watch_completables` or something to distinguish it from a list of children which are completable? See past comment about this list not being simply a list of completable descendants.